book/graphics.md

```diff
@@ -338,18 +338,32 @@ users to *scroll*\index{scroll} the page to look at different parts of it.
 
 ::: {.further}
 In English text, you can't wrap to the next line in the middle of a
-word (without hyphenation at least), but in Chinese that's mostly not a problem. Mostly, but not
-always! <span lang="zh">开关</span> means "button" but is composed of
-<span lang="zh">开</span> "on" and <span lang="zh">关</span> "off". A
-line break between them would be confusing, because you'd read "on
-off" instead of "button". The [ICU library][icu], used by both Firefox
-and Chrome, [uses dynamic programming][icu-wb] to guess phrase
-boundaries based on a [word frequency table][cjdict].
+word (without hyphenation at least), but in Chinese that's mostly not a problem.
+For example, <span lang="zh">开关</span> means
+"switch" but is composed of <span lang="zh">开</span> "on" and
```

Comment:
Google says "switch" is a better translation...

Reply:
My wife agrees